tools/esdlcmd/esdlcmd_common.hpp

```diff
@@ -103,7 +103,11 @@ typedef IEsdlCommand *(*EsdlCommandFactory)(const char *cmdname);
 #define ESDL_OPTION_ECL_INCLUDE_LIST    "--ecl-imports"
 #define ESDL_OPTION_ECL_HEADER_BLOCK    "--ecl-header"
 
-
+#define ESDLOPT_INCLUDE_PATH            "--includePath"
```

Comment:
in the name of consistency, eclcmd and esdlcmd normally use the format "--include-path".  Eclcc is a bit inconsistent.